src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -27178,35 +27178,41 @@ class Test2 : I1, I2, I3, I4
 }
 ";
             ValidateEventModifiers_17(source1,
-                // (4,41): error CS0180: 'I1.P1' cannot be both extern and abstract
-                //     abstract extern event System.Action P1; 
-                Diagnostic(ErrorCode.ERR_AbstractAndExtern, "P1").WithArguments("I1.P1").WithLocation(4, 41),
-                // (8,32): error CS0068: 'I2.P2': instance event in interface cannot have initializer
-                //     extern event System.Action P2 = null; 
-                Diagnostic(ErrorCode.ERR_InterfaceEventInitializer, "P2").WithArguments("I2.P2").WithLocation(8, 32),
```

Comment:
>Diagnostic(ErrorCode.ERR_InterfaceEventInitializer, "P2").WithArguments("I2.P2").WithLocation(8, 32), [](start = 16, length = 101)

Please restore original ordering of diagnostics so that it would be easy to see what diagnostics is removed/replaced/added.

Reply:
It looks like the list is still out of the original order.

---
In reply to: [354479092](https://github.com/dotnet/roslyn/pull/40011#discussion_r354479092) [](ancestors = 354479092)